test/functional/wallet_bumpfee.py

```diff
@@ -253,12 +253,16 @@ def test_dust_to_fee(self, rbf_node, dest_address):
     self.log.info('Test that bumped output that is dust is dropped to fee')
     rbfid = spend_one_input(rbf_node, dest_address)
     fulltx = rbf_node.getrawtransaction(rbfid, 1)
-    # size of transaction (p2wpkh, 1 input, 2 outputs): 141 vbytes
-    assert_equal(fulltx["vsize"], 141)
-    # bump with fee_rate of 0.00350000 BTC per 1000 vbytes
-    # expected bump fee of 141 vbytes * fee_rate 0.00350000 BTC / 1000 vbytes = 0.00049350 BTC
-    # but dust is dropped, so actual bump fee is 0.00050000
-    bumped_tx = rbf_node.bumpfee(rbfid, {"fee_rate": 0.0035})
+    # The DER formatting used by Bitcoin to serialize ECDSA signatures means signatures can have
+    # a variable size of 70-72 bytes, with most being 71 or 72 bytes. The signature in the witness
+    # is divided by 4 for the vsize, so this variance can take it across a vbyte boundary. Thus
```

Comment:
nit: this variance can take ~it~ _the weight_ across a 4-byte boundary.

The witness discount is applied to all bytes that don't appear in the non-segwit serialization:

- 1 byte (marker 0x00)
- 1 byte (flag 0x01)
- 1 byte (number of stack items 0x02)
- 71/72 bytes (sig length + signature 0x46..../0x47....)
- 34 bytes (pubkey length _ pubkey 0x21....)

TOTAL: 108/109 bytes weight => 27/28 bytes vsize

ref: https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#transaction-id

Reply:
good point, updated!